_oasis_remove_.ml

```diff
@@ -0,0 +1,7 @@
+open Printf
+
+let () =
+  let dir = Sys.argv.(1) in
+  (try Sys.chdir dir
+   with _ -> eprintf "Cannot change directory to %s\n%!" dir);
+  exit (Sys.command "ocaml setup.ml -uninstall")
```

Comment:
I fail to see the need for this as well.. `ocamlfind remove owl` should do the trick!?